pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/metadata.json

```
@@ -0,0 +1,53 @@
+{
+    "$schema":"https://google.github.io/security-research/kernelctf/metadata.schema.v1.json",
```

Switch to the latest schema (v2).
```suggestion
    "$schema":"https://google.github.io/security-research/kernelctf/metadata.schema.v2.json",
```